sdk/core/core-http/Changelog.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.8.10 - 2019-06-05
+- `axios` changed the way it treats properties of the request config in `0.19.0`. Previously we were setting `trasnformResponse` to `undefined`. This would indicate `axios` to not transform (`JSON.parse()`) the response body. In `0.19.0`, they are setting the default response transformer if transformResponse is set to `undefined`. This breaks our pasrsing logic where we are doing `JSON.parse()` on `operationResponse.bodyAsText`. Moreover, we are exposing the `bodyAsText` property in the generated clients. 
```

Comment:
While I find this interesting, this is too verbose for changelog IMO. Could just say, "implemented support for Axios 0.19.0.". If the work around is important for users to understand (even considering ourselves as users of this core package), could add "by setting the `transformResponse` property in the request config to an identity function that returns the response body as-is".

Reply:
I carried over this change from a [recent PR](https://github.com/Azure/ms-rest-js/pull/358) to `ms-rest-js` after we merged `@azure/core-http`, but I now realize that this file is confusing and should be cleared out.  I've created issue #3497 to track the remaining tasks for shipping `@azure/core-http`, this being one of them.